Protocol.md

```diff
@@ -726,25 +744,40 @@ RSocket resumption exists only for specific cases. It is not intended to be an 
 
 1. Resumption is optional behavior for implementations. But highly suggested. Clients and Servers should assume NO resumption capability by default.
 1. Resumption is an optimistic operation. It may not always succeed.
-1. Resumption is desired to be fast and require a minimum of state to be exchanged.
+1. Resumption is desired to be fast and requires a minimum of state to be exchanged.
 1. Resumption is designed for loss of connectivity and assumes client and server state is maintained across connectivity loss. I.e. there is no assumption of loss of state by either end. This is very important as without it, all the requirements of "guaranteed messaging" come into play.
 1. Resumption assumes no changes to Lease, Data format (encoding), etc. for resuming operation. i.e. A client is not allowed to change the metadata MIME type or the data MIME type or version, etc. when resuming operation.
 1. Resumption is always initiated by the client and either allowed or denied by the server.
 1. Resumption makes no assumptions of application state for delivered frames with respect to atomicity, transactionality, etc. See above.
 
+### Resume Agreement
+
+If communication with resumption support has been successfully initiated, the decision on whether a particular REQUEST should be resumed is made separately. For that purpose, a Requester set the (R)esume flag in the REQUEST(s) frame, and if the Responder responds in the first FRAME with the (R)esume flag set, then this stream is assumed to be resumable. Otherwise, if any of the sides did not set the (R)esume flag, the stream assumed to not be resumable. If a Requester has sent a REQUEST frame with the (R)esume flag set, but the connection is lost before the agreement has been made (e.g., any frames were received), such a REQUEST frame has to be redelivered unless the RESUME or RESUME_OK frame received after reconnection includes.
```

Comment:
seems to be missing the end of the sentence: "includes` the (R)esume flag`." I assume